src/Raven.Client/Documents/Smuggler/DatabaseItemType.cs

```diff
@@ -18,6 +18,7 @@ public enum DatabaseItemType
         LegacyDocumentDeletions = 1 << 9,
         LegacyAttachmentDeletions = 1 << 10,
         DatabaseRecord = 1 << 11,
-        Counters = 1 << 12
+        Counters = 1 << 12,
+        PendingClusterTransactions = 1 << 13
```

Comment:
just 'ClusterTransactions' ?

Reply:
That is actually pending ones, that haven't been applied yet